.gitignore

```diff
@@ -0,0 +1,19 @@
+**/.terraform/**
+**tfstate*
+**/*.pem
+terraform.tfvars
+**/account.json
+**/credentials
+.DS_Store
+.kitchen
+.terraform
+.terraform.tfstate.d
+# Gemfile has occasionally pinned versions of gems that shouldn't be included (e.g. Could not find aws-sdk-core-2.11.92 in any of the sources)
```

Comment:
`Gemfile.lock` should not be ignored. Any issues with Ruby dependency resolution should be addressed.